arduino-hal/Cargo.toml

```diff
@@ -6,15 +6,18 @@ edition = "2018"
 
 [features]
 default = ["rt"]
-rt = ["atmega-hal/rt"]
+rt = ["atmega-hal/rt", "attiny-hal/rt"]
```

Comment:
@Rahix I need help here. I don't know how I can switch `atmega-hal` with `attiny-hal`. Can you please share some hints?

Reply:
I think we're out of luck here :/ See https://github.com/rust-lang/cargo/issues/8832 ...

But does it maybe still work this way?  I know it is not optimal, but maybe it can still compile like this?

Reply:
hmm looks like without default `rt` I was able to compile. please review

Reply:
Solved in 8f0e0000d18d0fce3e6e3d8ab950b13a578d51cd :)